client/src/controllers/KeyboardController.ts

```diff
@@ -5,6 +5,12 @@ import Mousetrap from 'mousetrap';
 import 'mousetrap/plugins/global-bind/mousetrap-global-bind';
 
 import { WAGTAIL_CONFIG } from '../config/wagtailConfig';
+import { forceFocus } from '../utils/forceFocus';
+
+export enum KeyboardAction {
+  CLICK = 'click',
+  FOCUS = 'focus',
+}
 
 /**
  * Adds the ability to trigger a button click event using a
```

Comment:
I will add an example for focus usage in the examples when merging, also I found a typo where we were not using the value correctly. I can fix this up when merging.